rmd_files/strings.Rmd

```diff
@@ -272,10 +272,34 @@ stringr::str_replace_all(colours, "[^[A-Za-z0-9]]", "_")
 
 In the first example only the first match in each string has been replaced with an underscore, whereas in the second example all matches have been replaced.
 
+
 ---
 
 ### Exercise 1
 
+The column names of a table in an earlier exercise are: `offence_code`, `count_2016`, `count_2017`, `count_2018`, `count_2019`, `count_2020`.
+
+
+Using only functions covered in this part of the course, find the most efficient way to code this from scratch as:
```

Comment:
Wondering if the wording could do with being more precise here, to avoid confusion. For example:
`Using the functions covered in the 'Combining strings' section, find the most efficient way to define these column names as:`

Reply:
I wonder if that might make them search a bit too much... we could tell them exactly which functions to use, and just make the challenge be figuring out how to use them. The reason for writing that is that I wanted to avoid was the use of `rep()`, as this could arguably make it more efficient, albeit at the cost of making the explanation more confusing and getting into a debate about readability

- what do you think @chloepugh  ?

Reply:
```
Using only `stringr::str_c()` and `c()`, find the most efficient way to code this from scratch as:
a) A vector of strings
b) A single string, with column names separated by a comma and a space

Reply:
i opted for that, also bearing in mind your suggestion below